BookClass.java

```diff
@@ -0,0 +1,13 @@
+Public Class Book{
+
+public static void main(String[] args){
+
+String title;
+String author;
+double ISBN;
```

Comment:
Please add "private" modifiers for the attributes
```suggestion
private String title;
private String author;
private double ISBN;
```